### ISSUE TITLE
Problem during configuring nginx

### ISSUE THREAD

**Issue by deahbe** [chaosbunker/dockerbunker#6]
If I configure Wekan then I receive:
Generating self-signed certificate for wekan.domain.site [failed]

Restarting nginx container [failed]

2018/09/10 06:40:11 [warn] 84#84: the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /etc/nginx/conf.d/wekan.domain.site.conf:25
nginx: [warn] the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /etc/nginx/conf.d/wekan.domain.site.conf:25
2018/09/10 06:40:11 [emerg] 84#84: BIO_new_file("/etc/nginx/ssl/wekan.domain.site/cert.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/ssl/wekan.domain.site/cert.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
nginx: [emerg] BIO_new_file("/etc/nginx/ssl/wekan.domain.site/cert.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/ssl/wekan.domain.site/cert.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
nginx: configuration file /etc/nginx/nginx.conf test failed

→ `nginx -t` failed. Trying to add missing containers to dockerbunker-network.

Restarting nginx container [failed]

2018/09/10 06:40:12 [warn] 94#94: the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /etc/nginx/conf.d/wekan.domain.site.conf:25
nginx: [warn] the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /etc/nginx/conf.d/wekan.domain.site.conf:25
2018/09/10 06:40:12 [emerg] 94#94: BIO_new_file("/etc/nginx/ssl/wekan.domain.site/cert.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/ssl/wekan.domain.site/cert.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
nginx: [emerg] BIO_new_file("/etc/nginx/ssl/wekan.domain.site/cert.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/nginx/ssl/wekan.domain.site/cert.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
nginx: configuration file /etc/nginx/nginx.conf test failed

`nginx -t` failed again. Please resolve issue and try again.

What do I make wrong?

**Comment by chaosbunker** [comment]
I cannot reproduce your issue. What it says is that nginx cannot find the certificates where they should be. Make sure that you have permissions to write in the dockerbunker directories.

can you post the output of
`ls -la data/conf/nginx/ssl/wekan.domain.site`

This directory is created during setup and the certificates are generated right after that. Are you running `./dockerbunker` as `root`?

I would run `./dockerbunker` and then choose 'Wekan' > 'Destroy Wekan' and start over.

If you get stuck again try and manually create the certs and post the output.

First source dockerbunker.env
`source data/env/dockerbunker.env`

(next, make sure to replace wekan.domain.site with your domain)
```
openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout \
"${CONF_DIR}"/nginx/ssl/wekan.domain.site/key.pem -out \ 
"${CONF_DIR}"/nginx/ssl/wekan.domain.site/cert.pem -subj \
"/C=XY/ST=hidden/L=nowhere/O=Wekan/OU=IT \
Department/CN=wekan.domain.site"
```

**Comment by deahbe** [comment]
Do I need run dockerbunker.sh as root? Because I stored dockerbunker at my home directory I don't expect this and therefore I don't do it.
The directory wekan.domain.site (replaced by my own local domain) is empty. Logically.
I have followed your description to place an existing certificate in the directory data/conf/nginx/ssl. Inside the nginx container I had to remove "wekan.domain.site" from the path to the certificate and after that it has worked.
I have done this right after I wrote here, so I don't want to start from scratch now.
